documentation/business_care_intitule.md

```
@@ -0,0 +1,369 @@
+
+ATTENTION : Les document écrits doivent être fournis aux dates de suivi indiquées sur MyGes. 
+
+**ANNEXES** 
+
+***ANNEXE 1*** 
+
+**Grille tarifaire :** 
+
+
+
+<table><tr><th colspan="1">Starter </th><th colspan="2">Basic </th><th colspan="2">Premium </th></tr>
```

markdownlint-cli2 (0.17.2)

Inline HTML
Element: table

(MD033, no-inline-html)